### PR TITLE
bug 1562004. Add retry logic

### PR DESCRIPTION
**Pull request by jcantrill**
Additional changes to resolve https://bugzilla.redhat.com/show_bug.cgi?id=1562004

### CODE REVIEW

**Comment by jcantrill** [comment]
@richm Updated to reflect that 1.16.1 of the plugin was released to pull in:

https://github.com/uken/fluent-plugin-elasticsearch/pull/419
https://github.com/uken/fluent-plugin-elasticsearch/pull/418

File: fluentd/Dockerfile [hunk 1]
```diff
@@ -59,7 +48,7 @@ RUN yum-config-manager --enable rhel-7-server-ose-3.9-rpms && \
 
 ADD configs.d/ /etc/fluent/configs.d/
 ADD filter_k8s_meta_for_mux_client.rb /etc/fluent/plugin/
-ADD out_syslog_buffered.rb out_syslog.rb out_rawtcp.rb /etc/fluent/plugin/
+ADD out_syslog_buffered.rb out_syslog.rb /etc/fluent/plugin/
```
Comment (richm) [inline fluentd/Dockerfile]: you removed `out_rawtcp.rb`?

**Comment by richm** [comment]
this breaks MUX_CLIENT_MODE=minimal for the json-file case - probably due to the retagging - I think we may have to add `@label @OUTPUT` to all of the fluent plugin elasticsearch configs in order to skip straight to the output queue, and avoid all of the other processing.

**Comment by jcantrill** [comment]
/test json-file
flake?

**Pull request merged**